deployer/auth.py

```diff
@@ -89,12 +91,32 @@ def ensure_client(self, name, domains, connection_name):
             self._ensure_client_callback(client, domains)
 
         current_connections = self.get_connections()
+
+        if connection_name == 'password':
+            # Users should not be shared between hubs - each hub
+            # should have its own username / password database.
+            # So we create a new 'database connection' per hub,
+            # instead of sharing one across hubs.
+            db_connection_name = f'database-{name}'
```

Comment:
Wondering if there is a way to support the same user auth against several (but not all) of our hubs.
The use case I am thinking around (and avoid) is having several demo hubs and asking the user to sign up for each of them...

Reply:
Yeah, if we get to that we should be able to add an extra field here to set connection name. However, their user data will not transfer, making the idea of 'same user' tricky.

Reply:
> However, their user data will not transfer, making the idea of 'same user' tricky.

Yep, that's a good point.